tests/testsuite.src/configuration.at

```diff
@@ -409,7 +409,7 @@ name: "Empty Conf"
 ])
 
 # check if incomplete configuration result in error
-AT_CHECK([$COMPILE_ONLY -fmax-errors=134 -conf=test.conf prog.cob], [1], [],
+AT_CHECK([$COMPILE_ONLY -fmax-errors=135 -conf=test.conf prog.cob], [1], [],
```

Comment:
That's a strange change - shouldn't we just make that unlimited?

Reply:
Sure, we could. Actually the option used to be absent (defaults to 128) and I hit the limit at some point, so I had added this with "just the right limit". Let's go for unlimited.

Reply:
Actually, the option to make this number unlimited (by specifying 0) has been introduced by a later commit (5185) - don't know if I should try to merge it right away ?

Reply:
Yes, please.

Reply:
Actually, that makes too many conflicts (probably depends on earlier commits). I might just "cherry-pick" just the necessary changes to allow an unlimited error count...